src/embind/embind.js

```diff
@@ -1743,7 +1743,7 @@ var LibraryEmbind = {
                            '$releaseClassHandle'],
   $attachFinalizer: function(handle) {
     if ('undefined' === typeof FinalizationRegistry) {
-        attachFinalizer = function (handle) { return handle; };
+        attachFinalizer = (handle) => { return handle; };
```

Comment:
We don't actually use ES6+ syntax in Emscripten codebase, so it's better to revert to the way things were.

Reply:
@RReverser but you do! see https://github.com/emscripten-core/emscripten/commit/c0f1058cc53736a078e1bfaa42f0f2b8beebbccb

Reply:
Ah, 3 days ago. I guess things changed a lot during my 2 weeks OOO 😅

Reply:
A little oversight on my part when merging